File: c/chapter 3/power.cpp
//number raised to the power of anathor
#include<stdio.h>
#include<conio.h>
int main()
{
    int a,b,power=1;
    printf("enter number 1 =");
    scanf("%d",&a);
    printf("enter number 2 =");
    scanf("%d",&b);
    for(int i=1;i<=b;i++)
    {
     power=power*a;
    }
    printf("%d",power);
    getch();
}

    

File: java/power.java

import java.util.Scanner;
public class power{
public static void main(String args[])
{
int num1,num2,z;
System.out.println("enter the number 1");
Scanner in = new Scanner(System.in);
 
num1 = in.nextInt();
System.out.println("enter the number 2");
num2 = in.nextInt();
z=pow(num1,num2);
System.out.println("power num 1 raised to num 2  is = "+z);
}
public static  int pow(int a,int b)
{
int p=1;
for(int i=1;i<=b;i++)
    {
     p=p*a;
    }

return p;
}
}